aws/ondemand/facebook/hhvm/init-container.sh

```diff
@@ -32,6 +32,22 @@ if [ ${#DIRS[@]} != 1 ] || [ ! -d "${DIRS[0]}" ]; then
 fi
 BUILD_DIR="${DIRS[0]}"
 
+mkdir -p /root/.vscode-server/data/Machine/
+cat > /root/.vscode-server/data/Machine/settings.json <<ANALBUMCOVER
+{
+  "cmake.configureArgs": [
+    "-Wno-dev",
+    "-DMYSQL_UNIX_SOCK_ADDR=/var/run/mysqld/mysqld.sock",
+  ],
+  "cmake.defaultVariants": {
+      "buildType": {
+          "default": "reldeb",
```

Comment:
is this specific value required? If `RelWithDebInfo` is supported, I'd much strongly prefer that

Reply:
Yeah, I guessed they were equivalent - I was just hoping it would also accept the actual cmake value rather than something vscode-specific :(